### DISCUSSION TITLE
Regularizando situação no caso de prestação de serviço para fora

### DISCUSSION THREAD

**Discussion by AndreiCalazans** [DyegoCosta/trabalhando-remoto#184] [General]
Sou desenvolvedor de softwares. Gostaria de obter informações de vocês sobre a melhor maneira para eu regularizar a minha situação. Eu presto serviço para empresas da America do Norte, principalmente USA e Canada. No começo do ano como eu ainda prestava serviço apenas para o Brasil eu criei um MEI, porém meu rendimentos cresceram e agora estou exclusivamente trabalhando para fora. Meu problema é que não regularizei esta situação e estou recebendo o dinheiro via serviços de pagamentos online direto para minha conta física, meus clientes não requerem notas fiscais. Não tenho uma conta jurídica. 

Informações relevantes:

Atividade: Desenvolvimento de software.
Local de trabalho: Home office. 

Preciso saber qual caminho a perseguir e como lidar com esta situação que estou atualmente. Sei que preciso encerrar minha MEI e através de pesquisas na issues aqui sei que tenho 3 escolhas, 1) declarar como pessoa física através do Carne Leão (como eu pago mestrado para minha esposa e ela é uma dependente eu conseguiria ter deduções significativas), 2) Abrir uma nova empresa no regime Simples Nacional, 3) Abrir uma nova empresa no regime de Lucro Presumindo. 

Vi que tem varias pessoas na comunidade em situações parecidas e as informações sobre isso estão espalhadas nas issues, eu li tudo e cheguei a o resumo acima mais ainda não sei qual das opções escolher = O.

**Comment by renatonascalves** [comment]
Estou no mesmo barco que você. Atualmente, ainda sou MEI mas ano que vem já irei fazer a alteração. Segue o que meu contador recomendou.

Migrar de MEI para Simples Nacional. Receber pagamento na conta júrídica, que já possuo e a uso.

Não recomendaria sair do MEI agora, no meio do ano. Isso é errado, mas recomendaria declarar o ano de 2018 apenas com os 81k do limite. É possível declarar os ganhos para as empresas do exterior já que não é necessário emitir nota fiscal. O fato de ter recebido em conta pessoal, já que é MEI, não é necessariamente um problema agora.

O motivo: a multa que irá levar por ultrapassar o limite de 81k, além do propocional que ultrapassou, do MEI, irá lhe surpreender. Converse com um contador e peça uma projeção do custo de migração com os seus ganhos e você entenderá o que estou dizendo.

**

Quando estive no Simples, para diminuir a carga tributária, uma ideia é ao invés de receber via prolabore, receba como lucro da empresa, que é isenta de imposto na pessoa física.

As notas fiscais serão emitidas normalmente e você pagará os impostos de serviços, 6% no Recife.

Me parece que no sistema de Lucro Presumido, existe uma opção de diminuir os impostos de serviços para serviços prestados no exterior, mas não entendo bem. Isso foi algo que vi aqui no repositório, em outros _issues_.

Porém, meu conselho, converse com dois ou três contadores diferentes. Peça opiniões sobre qual é a melhor opção. A que terá um impacto menor nos impostos.

Abra uma conta empresarial. Banco do Brasill custa em torno de R$ 36 mensais. Atualmente, como conta MEI, custa R$ 17. Outros bancos nem cobram taxas.

**Comment by AndreiCalazans** [comment]
Interessante @renatonascalves, recebi uma recomendação similar de um contador local. 
Ainda estou ouvindo opniões antes de tomar uma decisão solida. Mas estou bem inclinado a seguir este caminho. 
Outro ponto interessante é que de acordo com o [portal do empreendedor](http://www.portaldoempreendedor.gov.br/duvidas-frequentes/12-quero-crescer-nao-sou-mais-mei-e-agora/12.1-o-que-ocorre-com-a-pessoa-que-estiver-enquadrada-na-lei-do-mei-e-estourar-o-faturamento-de-60-mil-anual), se exceder em até 20% os 81K no só pagamos uma DAS sobre a diferença, o que não pode é exceder dos 81K + 20% (97,2k), assim teríamos que recolher como simples. 

Valeu pelo comentário.

**Comment by copercini** [comment]
Pelo que sei são essas 3 alternativas mesmo:
1- Carnê leão vale a pena pelo "sossego", é só gerar, pagar e pronto, mas se prepare para pagar a tabela normal do IR, ou seja grande parte em 27,5%, pode ser calculado [aqui](http://www.receita.fazenda.gov.br/aplicacoes/atrjo/simulador/simulador.asp?tipoSimulador=M)

2 e 3- Ser exportador de serviços como empresa tem [isenções de impostos, COFINS, PIS/Pasep, IPI, ICMS e ao ISS](http://www.caubr.gov.br/wp-content/uploads/2017/03/APOSTILA-CORRETA-TRUBUTACAO-EXPORTACAO-DE-SERVICOS.pdf) tanto no simples como no lucro presumido. No simples se você conseguir cumprir algumas regras, consegue pagar apenas **3,05%** de imposto + IR e INSS sobre o pro labore, no final fica perto de **8%** , [aqui tem os cálculos e discussões de simples x lucro presumido](https://github.com/DyegoCosta/trabalhando-remoto/issues/54#issuecomment-355760630)

Já de adianto que abrir e manter empresa é muito burocrático, pra fazer qualquer coisa vc precisa registrar em cartório, mandar uma pilha de documentos, isso até pra abrir conta num banco, mas boa sorte

**Comment by samiraguiar** [comment]
@renatonascalves no SIMPLES também é possível diminuir os impostos. Atualmente estou pagando cerca de 3% de impostos mensais porque consegui isenção de PIS, COFINS e ISS por estar exportando serviço. A ideia é manter o pró-labore sempre em 28% do que você receber bruto, assim consegue se encaixar no anexo III (os dos 6%) e reduzir com as isenções.

**Comment by fforbeck** [comment]
@samiraguiar mas aí o que você tira no pró-labore fica dentro da isenção de PF? Ou paga menos IRPF? 
Só tentando entender se não acaba caindo no IRPF essa retirada e por consequência aumentando o imposto.

**Comment by copercini** [comment]
@fforbeck paga IRPF só do pro labore, que é 28% do valor
ou seja, do exemplo de R$15000, o pro labore seria R$4200, então o IRPR é só sobre R$4200, que dá R$308,87 apenas

**Comment by fforbeck** [comment]
Saquei @copercini, valeu!

**Comment by fabioluiz1** [comment]
@samiraguiar seu contador que emite a NF? https://github.com/DyegoCosta/trabalhando-remoto/issues/61#issuecomment-415418556

**Comment by fabioluiz1** [comment]
encontrei este passo-a-passo https://conube.com.br/blog/o-que-e-invoice, bastante sucinto e didático. alguém sabe dizer se é tranquilo fazer o registro no siscoserv?

**Comment by fabioluiz1** [comment]
@copercini obrigado!
Apesar das burocracia de manter uma empresa, abrir um CNPJ no Simples parece mesmo inevitável à medida que o faturamento aumenta.

**Comment by copercini** [comment]
@fabiolnm Fiquei em dúvida em relação ao siscoserv, vou pesquisar melhor sobre isso

**Comment by samiraguiar** [comment]
@fforbeck é isso que o @copercini disse, aí só precisa ficar atento ao faturamento pra manter o pró-labore dentro da faixa dos 28%.

@fabiolnm não, eu que faço. Na primeira ele me auxiliou, daí as outras estou fazendo sozinho. É pelo próprio sistema da prefeitura, coisa de 4 etapas bem simples.

Sobre o siscoserv, até onde sei empresas do SIMPLES e MEI são isentas (art. 2o, inciso I):
http://normas.receita.fazenda.gov.br/sijut2consulta/link.action?visao=anotado&idAto=38371

Aqui também fala um pouco disso:
https://www.siscoserv.net/quem-esta-dispensado-siscoserv

Só que no primeiro link, fiquei em dúvida sobre o trecho "Ficam dispensadas do registro [...] nas operações que não tenham utilizado **mecanismos de apoio ao comércio exterior** de serviços[...]". E no segundo, sobre "Caso não tenha recebido **benefícios**, o contribuinte estará dispensado nas seguintes hipóteses". O que será que são esses benefícios/mecanismos de apoio? Pela lógica, a isenção fiscal que recebemos (PIS, COFINS e ISS) deveria contar como "mecanismo de apoio". Será que é isso?

Edit: parece que encontrei:
http://www.mdic.gov.br/images/REPOSITORIO/scs/decin/Guia_B%C3%A1sico/Encarte_2_ao_Guia_B%C3%A1sico_para_a_exporta%C3%A7%C3%A3o_de_Servi%C3%A7os.pdf

**Comment by herodrigues** [comment]
@AndreiCalazans  trabalho há um tempo dessa mesma forma que você. Mesmo minha renda não sendo tão alta comparada com a sua, eu pago DARFs mensais e a porrada é grande. 
O que me deixa mais revoltado é que nossa "modalidade de trabalho" parece ser inexistente pra Receita Federal. Não podemos deduzir quase nada relacionados à nossa atividade. Fora isso, soma-se o que pagamos de impostos estaduais e municipais embutidos nas contas residenciais e o montante poder chegar a quase 50% da renda.

Eu já até pesquisei sobre abrir uma empresa, mas como o @copercini disse, a dor de cabeça é grande.
Infelizmente, nosso país está longe de ser um país livre economicamente falando.

**Comment by sservulo** [comment]
@herodrigues Abrir empresa no Brasil não é a coisa mais fácil do mundo, mas não é tão complicado assim, demorei cerca de um mês pra abrir a minha e a contadora fez boa parte do processo burocrático. No fim das contas tudo somou em torno de 1k para abertura da empresa e duas idas a uma empresa pra fazer o token digital (precisava para finalizar o processo de abertura com a prefeitura). Todo o resto consegui resolver de casa e com umas 2 viagens de motoboy. gg wp

**Comment by yuriploc** [comment]
@sservulo Quanto você pagou à contadora? Vai precisar de contador mensalmente? Isso deve entrar na conta também. Atualmente sou MEI e também estou avaliando qual a melhor opção a seguir
